transport-native-epoll/src/main/java/io/netty/channel/epoll/AbstractEpollStreamChannel.java

```diff
@@ -884,7 +884,6 @@ void epollInReady() {
 
                 if (close) {
                     shutdownInput();
-                    close = false;
```

Comment:
not sure I follow ... setting back to false seems unnecessary as we will not use it again